BlockEntity/BEFruitpress.cs

```diff
@@ -269,23 +274,11 @@ private void onTick100msServer(float dt)
             }
 
 
-            if (juiceableLitresLeft <= 0.01)
+            if (juiceableLitresLeft < 0.01)
             {
-                // Hack to fix rounding errors
-                if (cntBlock != null)
-                {
-                    float litres = cntBlock.GetCurrentLitres(BucketSlot.Itemstack);
-                    cntBlock.SetCurrentLitres(BucketSlot.Itemstack, (float)Math.Round(100 * litres) / 100f);
-                }
-
                 UnregisterGameTickListener(listenerId);
                 listenerId = 0;
 
-                int stacksize = GameMath.RoundRandom(Api.World.Rand, (float)juiceableLitresTransfered);
-                mashStack.Attributes.RemoveAttribute("juiceableLitresTransfered");
-                mashStack.Attributes.RemoveAttribute("juiceableLitresLeft");
-                mashStack.StackSize = (int)(stacksize * juiceProps.PressedDryRatio);
-

```

Comment:
This is moved to when the player removes the mash from the fruit press.  That makes it so that players can’t accidentally overwrite the dry mash inside with wet mash by adding more fruit, keeping the behavior of mash inside the machine more predictable.